common-services.yml

```diff
@@ -58,6 +59,8 @@ services:
       - shogun-client
       - shogun-admin
       - shogun-print
+    restart: unless-stopped
+
```

Comment:
```suggestion
```

Reply:
Good catch! Thank you! I will remove the line and then merge.